src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php

```diff
@@ -336,17 +350,48 @@ protected function castObject(Stub $stub, bool $isNested): array
             $parents[] = '*';
 
             $r = new \ReflectionClass($class);
+            $classAttributes = $r->getAttributes(AbstractAttribute::class, \ReflectionAttribute::IS_INSTANCEOF);
+
+            $properties = [];
+            foreach ($r->getProperties() as $property) {
```

Comment:
no way we call getProperties in cloners, this will kill perf + the code is broken because it doesn't get properties on parent classes